hip-0000-template.md

```diff
@@ -1,72 +1,491 @@
 ---
-hip: <HIP number (this is determined by the HIP editor)>
-title: <HIP title>
-author: <a list of the author’s or authors’ name(s) and/or username(s), or name(s) and email(s).>
-working-group: a list of the technical and business stakeholders' name(s) and/or username(s), or name(s) and email(s).
-type: <Standards Track | Informational | Process>
-category: <Core | Service | Mirror | Application>
-needs-council-approval: <Yes | No>
-status: <Draft | Review | Last Call | Active | Inactive | Deferred | Rejected | Withdrawn | Accepted | Final | Replaced>
-created: <date created on>
-discussions-to: <a URL pointing to the official discussion thread>
-updated: <comma separated list of dates>
-requires: <HIP number(s)>
-replaces: <HIP number(s)>
-superseded-by: <HIP number(s)>
+hip: TBD
+**title: Partitioning Functionality for Fungible Tokens on Hedera**
+author: Sam Wood <sam.wood@toko.network>, Stephanie Yi <stephanie.yi@toko.network>, Eleonora Odorizzi <eleonora.odorizzi@toko.network>, Vicky Lio <vicky.lio@toko.network>
+type: Standards Track
+needs-council-approval: Yes
+status: Idea
+created: 2023-06-07
+discussions-to:
+updated: 2023-06-07
 ---
 
 ## Abstract
 
-Please provide a short (~200 word) description of the issue being addressed.
+This HIP proposes a backwards compatible addition of partitioning[^1] functionality to Hedera Fungible Tokens, allowing token issuers to:
```

Comment:
```suggestion
This HIP proposes a backwards compatible addition of partitioning[^1] functionality to Hedera Fungible and Non-Fungible Tokens, allowing token issuers to:
```

This doesn't sound like it's only for fungible tokens. In the 'Motivation' section:

> The inability to perform bulk execution of native primitives on multiple NFTs limits scalability due to the computational cost of performing individual operations on a large number of NFTs

Reply:
Initially, we had also prepared a HIP for NFTs, we ended up not sending it. We will revise the motivation section accordingly, thank you for bringing it up.